scripts/platforms/greenhouse_apply_v2.py

```
@@ -0,0 +1,550 @@
+        return "", 0.0
+
+    try:
+        response = client.chat.completions.create(
+            model="llama-3.3-70b-versatile",
+            messages=[{"role": "user", "content": prompt}],
+            temperature=0.2,
+            max_tokens=260
+        )
+
+        raw = response.choices[0].message.content.strip()
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)